eng/create-announcement-draft.sh

```diff
@@ -120,4 +120,7 @@ else
   export RUNTIME_VERSION_SHORT="$RUNTIME_VERSION"
 fi
 
+echo "Tag URL: $TAG_URL"
```

Comment:
We can't do this - look at how the script is used - we use the output of this script for the announcement so these lines would appear in the official announcement.

Reply:
> Maybe we just check if `TAG_URL` is set?

I'm fine with that.

Reply:
We have looked at it in detail again and it seems this would only get printed when we use gist and we're doing 6.0/7.0. But we only use gists for 8.0 at this time so this would never get printed.

I don't like how it's just relying on the env variables so we decided to just remove these lines as the tag is linked in the announcement itself anyway